src/main/java/com/dao/homework/dao/ManufacturerDao.java

```diff
@@ -11,9 +11,7 @@
 
     Manufacturer update(Manufacturer manufacturer);
 
-    boolean delete(Manufacturer manufacturer);
-
-    boolean deleteById(Long manufacturerId);
+    boolean delete(Long manufacturerId);
 
     List<Manufacturer> getAllManufacturers();
```

Comment:
how about this comment?

Reply:
But i removed this method becuse it redundant and rename deleteById